scripts/backup/pg_backup.config

```diff
@@ -10,7 +10,7 @@ BACKUP_USER=
 HOSTNAME=
 
 # Optional username to connect to database as.  Will default to "postgres" if none specified.
-USERNAME=
+USR=
```

Comment:
Strange to me that we hardcoded a database below, but leave username empty. At the same time, as I can see, we cannot pass username via environments and for end-user there is only one way to set db user - update this file. I think we can do it right away and let this scripts be ready to use

Reply:
@vryazanov Actually it is ready for use, but will use postgres user

Reply:
@krzyk I'm sure it works when postgres and app on the same machine, but not sure there is the same configuration on staging/production. So I think this point will wait for the next review by @emilianodellacasa

Reply:
@krzyk @vryazanov Db and app will not be on the same server, please take that into account

Reply:
@vryazanov @emilianodellacasa Server location is irrelevant for username (thats why we have HOSTNAME variable) hardcoding USR to be the same as production user is also not a good idea. So I don't know what I should do here.